jastrow.py

```diff
@@ -201,63 +258,100 @@ def updateinternals(self,e,epos,mask=None):
         #update b and c sums. This overlaps with testvalue()
```

Comment:
We removed the overlap.

Reply:
Taken out. Also changed the Jastrow with spin's updateinternals docstring to say a, b, and c sums.